docs/docs/installation/index.md

```diff
@@ -0,0 +1,44 @@
+# Install Mathesar
+
+This page contains instructions to install Mathesar on various platforms.
+
+## Guided installation using our install script
+- Use our install script to guide you install Mathesar via a series of prompts:
+    ```sh
+    bash <(curl -sfSL https://raw.githubusercontent.com/centerofci/mathesar/0.1.1/install.sh)
+    ```
+- The script sets up Mathesar using Docker Compose and configures the admin user [under the hood](./guided-install/under-the-hood.md).
+- !!! warning "Limitations"
+    This is a convenient way to install Mathesar. However, it requires `sudo` privileges (admin access), and sets up a limited set of configuration options. Use the [Docker Compose installation](./docker-compose/index.md) option to exert more control.
+- [Read detailed instructions](./guided-install/index.md)
+
+## Docker Compose 
+- Download our [docker-compose.yml](https://github.com/centerofci/mathesar/raw/master/docker-compose.yml) file.
+    ```sh
+    wget https://github.com/centerofci/mathesar/raw/master/docker-compose.yml
+    ```
+- Configure [environment variables](../configuration/env-variables.md).
+    ```sh
+    # Sample .env file
+    wget https://github.com/centerofci/mathesar/raw/master/.env.example
+    mv .env.example .env
+    ```
+- Start Mathesar and create the super user.
+    ```sh
+    docker compose -f docker-compose.yml up -d
+    docker exec -it mathesar_service python manage.py createsuperuser
+    ```
+- [Read detailed instructions](./docker-compose/index.md)
```

Comment:
This is better, since it doesn't have the box above to break up the section. It'd still be a bit clearer what to expect if the text was something like 'Detailed Docker Compose Instructions'.

Reply:
Fixed in https://github.com/centerofci/mathesar/commit/036f1745cc11277ec055740ab8a91505ccdf4c55